interceptor/tracing_interceptor.go

```diff
@@ -886,16 +1009,41 @@ func (t *tracingInterceptor) writeSpanToHeader(span TracerSpan, header map[strin
 	return nil
 }
 
-func newErrFut(ctx workflow.Context, err error) workflow.ChildWorkflowFuture {
+func (t *tracingInterceptor) writeSpanToNexusHeader(span TracerSpan, header nexus.Header) error {
+	// Serialize span to map
+	data, err := t.tracer.MarshalSpan(span)
+	if err != nil || len(data) == 0 {
+		return err
+	}
+	// Put on header
+	for k, v := range data {
+		header.Set(k, v)
+	}
+	return nil
+}
+
+func (t *tracingInterceptor) readSpanFromNexusHeader(header nexus.Header) (TracerSpanRef, error) {
+	// Ignore unmarshal errors, e.g. when an expected header isn't set.
+	// This works differently from non Nexus headers where the trace context is serialized entirely into a single
+	// payload and the presence of that payload implies a valid header.
+	span, _ := t.tracer.UnmarshalSpan(header)
```

Comment:
Per off-PR discussion, let's see if we can make `UnmarshalSpan` calls in the tracing impls return `nil` if the proper header is not present (already done for datadog, OpenTracing may have a certain error, and have to hardcode "traceparent" lookup for OTel)